README.md

```diff
@@ -1,192 +1,74 @@
-8. -overwrite=true/false : Whether to overwrite the current model data under 'modelPath'. Default is false. You may need to set to true to avoid IOException when saving the model to a path already exists.
-9. -hasHeader=true/false : Indicate if your csv file has header.
+4. `-trainDataPath=[path]`: Path to your training data file(s), required when mode is NOT 'transform'.
+5. `-trainEvalDataPath=[path]`: Path to your data file(s) for training with evaluation. Optional.
+6. `-evalDataPath=[path]`: Path to your test(evaluation) data file(s), required when mode is NOT 'train'.
+7. `-modelPath=[path]`: Path to save model after training, or where to load model for transforming only. Required only when mode is 'transform'.
+8. `-overwrite=[true|false]`: Whether to overwrite the current model data under 'modelPath'. Default is false. You may need to set to true to avoid IOException when saving the model to a path already exists.
+9. `-hasHeader=[true|false]`: Indicate if your csv file has header.
 
+# Other Examples
+There are three example apps included in this repo.
 
-## <a name="jupyternotebook"></a> jupyter notebook
+- [Mortgage](/src/main/scala/ai/rapids/spark/examples/mortgage) 
+- [Taxi](/src/main/scala/ai/rapids/spark/examples/taxi)
+- [Agaricus](/src/main/scala/ai/rapids/spark/examples/agaricus)
 
-Make sure you have jupyter notebook installed.
+Each example requires a different dataset. If you would like to produce your own dataset, the code used to prepare each sample is included in this repo (please see [Preparing Datasets](docs/preparing_datasets.md)).
 
-Install Toree:
-```bash
-pip install toree
-```
+## Download Trainable Dataset
 
-install scala kernel with Toree:
-```bash
-jupyter toree install --spark_home=$SPARK_HOME --spark_opts='--master=spark://<YOUR_IP:PORT> \
---executor-memory 32G \
---jars /data/spark/libs/cudf-0.8-SNAPSHOT-cuda10.jar,/data/spark/libs/xgboost4j-0.90-SNAPSHOT.jar,/data/spark/libs/xgboost4j-spark-0.90-SNAPSHOT.jar' \
---user \
---kernel_name=<YOUR_KERNEL_NAME>
+You can get a small size transformed dataset for each example class in `datasets` folder in this repo: 
 
-```
+1. [Mortgage Dataset(csv)](https://github.com/rapidsai/spark-examples/blob/master/datasets/mortgage-small.tar.gz?raw=true)
+2. [Taxi Dataset(csv)](https://github.com/rapidsai/spark-examples/blob/master/datasets/taxi-small.tar.gz?raw=true)
+3. [Agaricus(csv)](https://github.com/rapidsai/spark-examples/blob/master/datasets/agaricus.tar.gz?raw=true)
 
-Then you start your notebook and open [`mortgage-gpu.ipynb`](https://github.com/rapidsai/spark-examples/blob/master/notebook/mortgage-gpu.ipynb) to explore.
+Please note that the data above is only provided for convenience to get started with each example. In order to test for performance, please prepare a larger dataset by following [Preparing Datasets](docs/preparing_datasets.md).
```

Comment:
we have a link to the larger mortgage dataset right, perhaps we could also add that here

Reply:
we have it in each of the starting guides, but will add here too